cool_seq_tool/sources/transcript_mappings.py

```diff
@@ -7,7 +7,17 @@
 
 
 class TranscriptMappings:
-    """The transcript mappings class."""
+    """Provide mappings between gene symbols and RefSeq + Ensembl transcript accessions.
+
+    Uses ``LRG_RefSeqGene`` and ``transcript_mappings.csv``, which will automatically
+    be acquired if they aren't already available. See the
+    :ref:`configuration <configuration>` section in the documentation for information
+    about manual acquisition of data.
+
+    In general, this class's methods expect to receive NCBI gene symbols, so users
+    should be careful about the sourcing of their input in cases where terms are
+    conflicted or ambiguous.  # TODO is that true?
```

Comment:
I think that's right. We should come back to this. I think we swap between HGNC and NCBI. We removed the gene normalizer. Maybe we should add it back for this

Reply:
#232